### DISCUSSION TITLE
ha-floorplan card not available following removal and reinstallation

### DISCUSSION THREAD

**Discussion by m90att** [ExperienceLovelace/ha-floorplan#239] [Q&A] [answered]
I previously had ha-floorplan installed (via HACS) but didn't find the time to use it to its full potential.  When slimming down my setup I removed it (via HACS) and all was fine.  I know want to dedicate more time to giving it a better go and I installed it again, via HACS, but the card does is not visible for adding to my dashboard.  I've tried uninstalling and reinstalling, the resource is correctly configured, and I've restarted at the correct points.  

I can find nothing in the logs and the dev console shows that it's installed:
![image](https://user-images.githubusercontent.com/56609577/191733057-db096f59-571e-4a72-a1d9-a64cb6b1f149.png)

Can anyone advise what I can do to see the card please?

Thanks, Matt

Home Assistant 2022.9.6
Supervisor 2022.09.1
Operating System 8.5
Frontend 20220907.2 - latest

**Comment by OzGav** [comment, marked as answer]
What do you mean by this `the card does is not visible for adding to my dashboard`. If you are trying to add it via the UI this is not possible. You must configure via yaml only

**Reply by m90att** [reply]
ah, thank you!
